lsp/src/Language/LSP/VFS.hs

```diff
@@ -120,6 +151,7 @@ data VfsLog
 instance Pretty VfsLog where
   pretty (SplitInsideCodePoint pos r) =
     "VFS: asked to make change inside code point. Position" <+> viaShow pos <+> "in" <+> viaShow r
+  pretty ClosedVirtualFile = "VFS: trying to handle closed virtual file"
```

Comment:
It's much more specific than that: we tried to apply a _change_ to a closed file. We should include the URI.